### PR TITLE
Enhancement add countries exceptions billing and shipping

### PR DESCRIPTION
**Pull request by matheuscl**
Fix issue #10436.
I think the Title can be improved for both fields, any sugestions?

### CODE REVIEW

File: includes/admin/settings/class-wc-settings-general.php [hunk 1]
```diff
@@ -75,6 +75,15 @@ public function get_settings() {
 			),
 
 			array(
+				'title'   => __( 'Sell to all Countries except', 'woocommerce' ),
```
Comment (mikejolley) [inline includes/admin/settings/class-wc-settings-general.php]: I'd reword these to 'Exclude specific countries", and move it after the 'sell to' option.

File: assets/js/admin/settings.js [hunk 2]
```diff
@@ -3,10 +3,15 @@
 
 	// Countries
 	$( 'select#woocommerce_allowed_countries, select#woocommerce_ship_to_countries' ).change( function() {
-		if ( $( this ).val() === 'specific' ) {
-			$( this ).parent().parent().next( 'tr' ).show();
-		} else {
+		if ( 'disabled' === $( this ).val() ) {
+			$( this ).parent().parent().next( 'tr' ).hide();
+			$( this ).parent().parent().next().next( 'tr' ).hide();
```
Comment (mikejolley) [inline assets/js/admin/settings.js]: Wouldn't closest() be cleaner?

Comment (matheuscl) [inline assets/js/admin/settings.js]: Yes, i talk with @claudiosmweb about that, i'll make some changes in this PR =)

**Comment by matheuscl** [comment]
Talking with @claudiosmweb we decided the best approach for this issue is creating a new `option` to `woocommerce_allowed_countries` called like "Sell to all countries except", with this new option we can remove the `woocommerce_all_except_ship_to_countries` and let the option to exclude only in "Sell to", because the actual fix is a little confusing.
